### PR TITLE
Added Quality indicator tooltip

### PR DESCRIPTION
**Pull request by adamzerella**
Added in new 'help' logo and tooltip

***IGNORE THE BRANCH NAME, THIS RELATES TO ISSUE #518 ONLY.***

### Checklist
- [X] I've updated CHANGES.md with what I changed.
- [X] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column

### CODE REVIEW

File: magda-web-client/src/UI/QualityIndicator.js [hunk 1]
```diff
@@ -34,7 +37,14 @@ function QualityIndicator(props) {
                 {qualities.map((q, i) => (
                     <span key={i} style={getBarColor(i)} />
                 ))}
-            </span>
+            </span>&nbsp;
+            <div className="tooltip">
+                <img src={helpIcon} alt=""></img>
+                    <span className="tooltiptext">
+                        Calculated using the <a href="page/dataset-quality">Open Data scale</a>
```
Comment (AlexGilleran) [inline magda-web-client/src/UI/QualityIndicator.js]: You'll need to use the react-router `<Link>` - look at `ProjectDetails.js` :)

File: magda-web-client/src/UI/QualityIndicator.js [hunk 2]
```diff
@@ -34,7 +37,14 @@ function QualityIndicator(props) {
                 {qualities.map((q, i) => (
                     <span key={i} style={getBarColor(i)} />
                 ))}
-            </span>
+            </span>&nbsp;
+            <div className="tooltip">
+                <img src={helpIcon} alt=""></img>
```
Comment (AlexGilleran) [inline magda-web-client/src/UI/QualityIndicator.js]: Set the alt to "Help Link" or something maybe?

File: magda-web-client/src/UI/QualityIndicator.js [hunk 3]
```diff
@@ -34,7 +38,14 @@ function QualityIndicator(props) {
                 {qualities.map((q, i) => (
                     <span key={i} style={getBarColor(i)} />
                 ))}
-            </span>
+            </span>&nbsp;
+            <div className="tooltip">
+                <img src={helpIcon} alt="Help Link"></img>
+                    <span className="tooltiptext">
+                        Calculated using the <Link to="page/dataset-quality">Open Data scale</Link>
```
Comment (AlexGilleran) [inline magda-web-client/src/UI/QualityIndicator.js]: Looks good - just make it `/page/dataset-quality` instead of `page/dataset-quality` :)

**Comment by adamzerella** [comment]
@AlexGilleran Modified the `<Link>` location. If it's possible I'd like to close this PR now, assuming its fine so I can use the branch name again for #575 (Not a big deal but I just want to tidy up)

**Pull request merged**